aredis_om/model/model.py

```diff
@@ -765,6 +765,10 @@ def sort_by(self, *fields: str):
             return self
         return self.copy(sort_fields=list(fields))
 
+    async def page(self, offset, count):
```

Comment:
We will need to add at least one test for this to the test suites for the Hash and JSON models.

Hash tests: https://github.com/redis/redis-om-python/blob/main/tests/test_hash_model.py

JSON tests: https://github.com/redis/redis-om-python/blob/main/tests/test_json_model.py